scripts/webpack/toolbox.config.js

```diff
@@ -56,6 +56,26 @@ module.exports = {
     new webpack.DefinePlugin({
       __PRODUCTION__: true
     })
+    // new CircularDependencyPlugin({
```

Comment:
This was helpful for debugging why I was having difficulty with a webpack build at one point. Decided to leave the full config in here if it helps anybody else in the future